package.json

```diff
@@ -116,6 +116,7 @@
     "css-loader": "^0.28.4",
     "del": "^3.0.0",
     "devtron": "^1.4.0",
+    "electron": "^2.0.0-beta.4",
```

Comment:
A ha, I just degrade electron to 2.0.0-beta.1, this change maybe conflict with the last commit I just push.